.github/workflows/phpcs.yml

```diff
@@ -21,7 +21,7 @@ jobs:
           export PHPCS_DIR=/tmp/phpcs
           export SNIFFS_DIR=/tmp/sniffs
           git clone -b master --depth 1 https://github.com/squizlabs/PHP_CodeSniffer.git $PHPCS_DIR
-          git clone -b master --depth 1 https://github.com/WordPress-Coding-Standards/WordPress-Coding-Standards.git $SNIFFS_DIR
+          git clone -b 2.3.0 --depth 1 https://github.com/WordPress-Coding-Standards/WordPress-Coding-Standards.git $SNIFFS_DIR
```

Comment:
Looks like they changed 'master' to 'main'. This would be easier to maintain if we can avoid hardcoding a version number.

Reply:
Thanks! Can you please fix that error so we don't get stuck on v2.x? We can fix it in a different PR.

Reply:
For this PR, we can switch it back to 2.x like you had it. Then we can still get it merged and not hold up other PRs.